GVRf/Framework/src/org/gearvrf/GVRSceneObject.java

```diff
@@ -483,6 +483,42 @@ public GVREyePointeeHolder getEyePointeeHolder() {
     }
 
     /**
+     * Simple, high-level API to enable or disable eye picking for this scene
+     * object.
+     * 
+     * The {@linkplain #attachEyePointeeHolder(GVREyePointeeHolder) low-level
+     * API} gives you a lot of control over eye picking, but it does involve an
+     * awful lot of details. Since most apps are just going to use the
+     * {@linkplain #attachEyePointeeHolder() simple API} anyhow, this method
+     * (and {@link #getPickingEnabled()}) provides a simple boolean property.
+     * 
+     * @param enabled
+     *            Should eye picking 'see' this scene object?
+     * 
+     * @since 2.0.2
+     */
+    public void setPickingEnabled(boolean enabled) {
+        if (enabled != getPickingEnabled()) {
+            if (enabled) {
+                detachEyePointeeHolder();
+            } else {
+                attachEyePointeeHolder();
```

Comment:
> I think the logic is reverse here. I mean if enabled we do attach otherwise we do detach.

You're right! When I added the 'if-any-change' test, I mixed up the 'current state' and the 'desired state'.